app/frontend/types.ts

```diff
@@ -55,7 +55,19 @@ export type _XPubKey = {
   xpubHex: HexString
 }
 
-export type AuthMethod = '' | 'hw-wallet' | 'mnemonic' // TODO
+export enum AuthMethodEnum {
```

Comment:
AuthMethodType would imho be a more suitable name - "enum" is a low-level concept not really worth exposing via naming